wasm/js/webrtc.js

```diff
@@ -109,6 +181,26 @@
 			}
 		},
 
+		rtcLocalDescriptionSdp: function(pc) {
+			if(!pc) return 0;
+			var peerConnection = WEBRTC.peerConnectionsMap[pc];
+			var localDescription = peerConnection.localDescription;
+			if(!localDescription) return 0;
+			var sdp = WEBRTC.allocUTF8FromString(localDescription.sdp);
+			// sdp should be freed later in c++.
+			return sdp;
+		},
+
+		rtcLocalDescriptionType: function(pc) {
```

Comment:
Same, could you please rename `rtcLocalDescriptionType` to `rtcGetLocalDescriptionType`?